packages/discovery-provider/plugins/pedalboard/apps/solana-relay/src/utils/transaction.ts

```diff
@@ -110,12 +110,11 @@ export const sendTransactionWithRetries = async ({
     if (!sendOptions?.skipPreflight) {
       const simulatedRes = await connection.simulateTransaction(transaction)
       if (simulatedRes.value.err) {
-        // @ts-ignore Typescript is confused about deps
         throw new SendTransactionError({
           action: 'simulate',
           signature: confirmationStrategy.signature,
           transactionMessage: JSON.stringify(simulatedRes.value.err),
-          logs: simulatedRes.value.logs
+          logs: simulatedRes.value.logs ?? undefined
```

Comment:
Is this meant to specifically coerce `null` -> `undefined`? Otherwise seems unnecessary.